packages/controllers/helpers.coffee

```diff
@@ -0,0 +1,2 @@
+Template.registerHelper 'paragraphs', (text) ->
```

Comment:
Since there's only one template that needs this now, it doesn't have to be a global helper.